app/views/exam_templates/view_logs.html.erb

```diff
@@ -1,5 +1,11 @@
 <% content_for :title, "#{@assignment.short_identifier} #{SplitPdfLog.model_name.human.pluralize}" %>
 
+<% if flash[:split] %>
+    <script type="text/javascript">
+        poll_job("<%= flash[:split] %>", undefined, function() {window.location.reload()});
```

Comment:
You can just pass in `window.location.reload` (without calling it); you don't need to wrap it in an anonymous function.